test/fixtures/dialects/snowflake/alter_table.sql

```diff
@@ -51,3 +51,7 @@ ALTER TABLE "my_table" ALTER COLUMN "my_column" SET MASKING POLICY my_masking_po
 ALTER TABLE SAMPLE_DB.SAMPLE_SCHEMA.TBL UNSET COMMENT;
 
 ALTER TABLE table1 UNSET COMMENT, DATA_RETENTION_TIME_IN_DAYS;
+
+ALTER TABLE table1 ADD CONSTRAINT constraint1 FOREIGN KEY (address) REFERENCES addresses (address) ON DELETE RESTRICT ON UPDATE CASCADE;
+
+ALTER TABLE table2 ADD CONSTRAINT constraint2 FOREIGN KEY (address) REFERENCES addresses (address) ON DELETE NO ACTION ON UPDATE SET NULL;
```

Comment:
Can you add at least one `MATCH` example to the unit tests?

Reply:
Thanks for the review @WittierDinosaur . I've update the PR with additional test case